.gitignore

```diff
@@ -1 +1,3 @@
 ﻿OpenShift/helm/nexus-iq/myvalues.yaml
+.idea/*
```

Comment:
could this just be `.idea` so it's not tempted to even get the directory?

Reply:
I used a tool's defaults but i can make this better  👍

Reply:
FWIW Git doesn't track directories